Mage.Client/src/main/java/mage/client/game/GamePanel.java

```diff
@@ -305,10 +305,14 @@ public void cleanUp() {
         this.players.clear();
         this.playersWhoLeft.clear();
 
-        jLayeredPane.remove(abilityPicker);
+        if (jLayeredPane!= null) {
+            jLayeredPane.remove(abilityPicker);
+        }
         this.abilityPicker.cleanUp();
 
-        jLayeredPane.remove(DialogManager.getManager(gameId));
+        if (jLayeredPane != null) {
```

Comment:
minor: couldn't this `jLayeredPane.remove(DialogManager.getManager(gameId));` be moved up to the other conditional checking on `jLayeredPane!= null` on line 308?